conformance/tests/test_published_numbers_provenance.py

```
@@ -0,0 +1,335 @@
+            kind, extra = chk.classify_measured_commit(first, ["deleted.txt"], repo)
+            self.assertEqual(kind, chk.DIRTY)
+            self.assertEqual(extra, ["deleted.txt"])
+
+    def test_one_malformed_dependency_is_not_filtered_away(self):
+        """A valid dirty path next to pathspec magic is still a typed finding."""
+        with tempfile.TemporaryDirectory() as raw:
+            repo = Path(raw)
+            first = self._tiny_repo(
+                repo, {"tracked.txt": "a\n"}, later={"tracked.txt": "b\n"}
+            )
+            kind, extra = chk.classify_measured_commit(
```

Ruff (0.16.1)

[warning] Unpacked variable `extra` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)